ios/FluentUI/Navigation/Shy Header/ShyHeaderController.swift

```diff
@@ -124,6 +124,12 @@ class ShyHeaderController: UIViewController {
         updateNavigationBarStyle()
     }
 
+    // This is needed as a safety measure to ensure the colors are updated with the correct theme
+    override func viewDidAppear(_ animated: Bool) {
```

Comment:
Is there any lag in updating the color? Like, is there a flash of comm blue before going to the correct color for the theme?

Reply:
There is no visual lag because when we back out of a file the ShyHeaderController's view is hidden behind the NavigationBar, its background view and the shy header view. It only gets pushed up when searching.